ansible/server-state-playbooks/nightshade-web/playbook.yml

```diff
@@ -0,0 +1,184 @@
+          [Service]
+          Environment="PYTHONPATH=$PYTHONPATH:/opt/omero/web/web-extensions"
+      notify:
+        - reload systemd
+
+  roles:
+
+    # Root LV Size 
+    - role: openmicroscopy.lvm-partition
+      lvm_lvname: "{{ provision_root_lvname }}"
+      lvm_vgname: "{{ provision_root_vgname }}"
+      lvm_lvmount: /
+      lvm_lvsize: "{{ provision_rootsize }}"
+      lvm_lvfilesystem: "{{ provision_root_filesystem }}"
+
+    # OMERO.web configuration in host_vars in different repository
+    - role: openmicroscopy.omero-web
+      omero_web_release: 5.2.8
+
+    - role: openmicroscopy.system-monitor-agent
+      when: >
+            ((ansible_env.COBBLER_SERVER is defined)
+            and (ansible_env.COBBLER_SERVER == "spacewalk.lifesci.dundee.ac.uk"))
+
+  post_tasks:
+    - name: Install open-vm-tools if system is a VMware vm
+      become: yes
+      yum:
+        name: open-vm-tools
+        state: latest
+      when: >
+           ((ansible_virtualization_type is defined)
+           and (ansible_virtualization_type == "VMware"))
+
+    - name: NGINX - SSL File Deployment - prepare directory
+      become: yes
+      file:
+        path: "{{ nginx_ssl_files_path }}"
+        state: directory
+        owner: root
+        group: root
+        mode: "u=r,go=" 
+
+    - name: NGINX - SSL File Deployment
+      become: yes
+      copy:
+        dest="{{ item.key }}"
+        content="{{ item.value.content }}"
+        owner="{{ item.value.owner }}"
+        group="{{ item.value.group }}"
+        mode="{{ item.value.mode }}"
+      with_dict: "{{ nginx_ssl_cert_files }}"
+      no_log: true
+
+    # post 2.3 'destfile' should be renamed 'path'
+    - name: NGINX - SSL Configuration - Additional listen port
+      become: yes
+      lineinfile:
+        destfile: /etc/nginx/conf.d/omero-web.conf
+        insertafter: '    listen 80;'
+        line: '    listen 443 ssl;'
+            
+    # post 2.3 'destfile' should be renamed 'path'
+    - name: NGINX - SSL Configuration - Rest of SSL section to omero-web.conf
+      become: yes
+      blockinfile:
+        destfile: /etc/nginx/conf.d/omero-web.conf
+        insertbefore: '.*sendfile.*'
+        block: |2+
+
+              ssl_certificate {{ nginx_ssl_files_path }}/{{ nginx_ssl_cert_filename }};
+              ssl_certificate_key {{ nginx_ssl_files_path }}/{{ nginx_ssl_key_filename }};
+              ssl_protocols  {{ nginx_ssl_protocols }}
+            
+              if ($ssl_protocol = "") {
+                  rewrite ^/(.*) https://$host/$1 permanent;
+              }
+      notify:
+        - restart nginx
+
+    # 'manual' install of omero.web.apps for Nightshade feature parity
+
+    # note: system user var defined in openmicroscopy.omero-web as a default
+    - name: OMERO.web apps | top-level folder
+      become: yes
+      file:
+        path: "{{ omero_web_extensionsdir }}"
+        state: directory
+        owner: "{{ omero_web_system_user }}" 
```

Comment:
Make it owned by root unless you have reason to believe OMERO.web needs writes access to its code.